source/parsed_kdtree_distance.cc

```diff
@@ -51,8 +51,11 @@ double ParsedKDTreeDistance<dim>::value(const Point<dim> &p, const unsigned int
   kdtree->findNeighbors(resultSet, &p[0], nanoflann::SearchParams());
   //index.knnSearch(query, indices, dists, num_results, mrpt_flann::SearchParams(10));
 
+  // silence a warning about unused variable
+  // when compiling in release mode
+  (void)component;
```

Comment:
Because this is the implementation of a virtual function of the base class `dealii::Function`.